packages/react-devtools-shared/src/backend/utils.js

```diff
@@ -220,11 +221,72 @@ export function formatWithStyles(
   }
 }
 
+// NOTE: KEEP IN SYNC with src/hook.js
+// Skips CSS and object arguments, inlines other in the first argument as a template string
+export function formatConsoleArguments(
+  maybeMessage: any,
+  ...inputArgs: $ReadOnlyArray<any>
+): $ReadOnlyArray<any> {
+  if (inputArgs.length === 0 || typeof maybeMessage !== 'string') {
```

Comment:
maybe it is worth checking here if "maybeMessage" is an ANSI template.

Reply:
> maybe it is worth checking here if "maybeMessage" is an ANSI template.

Should be safe if this is correct ANSI template, it should override the stylings applied by React DevTools:
![Screenshot 2024-06-17 at 14 13 04](https://github.com/facebook/react/assets/28902667/aaf44a10-54ed-433a-a3ac-f5d16ddbbe60)